src/components/admonition/admonition.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, h, Prop } from '@stencil/core';
+import { AdmonitionType, getHeading, getIcon } from './admonition.types';
+
+/**
+ * Admonitions, also known as call-outs, very useful for including supportive content
+ * inside a user interface, or within a large piece of text.
+ *
+ * When used in a document or text based user interface, the admonitions attract
+ * reader's attention to a particular piece of information,
+ * without significantly interrupting their flow of reading the document.
+ *
+ * On a user interface, their usage will be experienced more intrusively by
+ * end-users. But they could be good choices when you indent to slightly
+ * disturb users' attention, and challenge them to pay extra attention
+ * to the information presented. In such cases, admonitions should not
+ * be used as static and constantly present elements of a UI. They should
+ * rather be displayed when something unusual or remarkable demands
+ * users attention.
+ *
+ * @exampleComponent limel-example-admonition-info
+ * @exampleComponent limel-example-admonition-warning
+ * @exampleComponent limel-example-admonition-danger
+ * @exampleComponent limel-example-admonition-note
+ * @exampleComponent limel-example-admonition-tip
+ * @exampleComponent limel-example-admonition-success
+ * @exampleComponent limel-example-admonition-failure
+ * @exampleComponent limel-example-admonition-example
+ * @exampleComponent limel-example-admonition-quote
+ * @exampleComponent limel-example-admonition-question
+ * @exampleComponent limel-example-admonition-custom-heading
+ * @exampleComponent limel-example-admonition-custom-component
+ * @exampleComponent limel-example-admonition-styles
+ * @exampleComponent limel-example-admonition-composite
+ */
+@Component({
+    tag: 'limel-admonition',
+    shadow: true,
+    styleUrl: 'admonition.scss',
+})
+export class Admonition {
+    /**
+     * Title of the admonition
+     */
+    @Prop({ reflect: true })
+    public heading?: string;
+
+    /**
+     * Description of the admonition
+     */
+    @Prop({ reflect: true })
+    public description?: string;
+
+    @Prop({ reflect: true })
+    public type?: AdmonitionType = 'info';
+
+    public render() {
+        return [
+            <div class="side">
+                <limel-icon name={getIcon(this.type)} />
+            </div>,
+            <div class="main">
+                <h1 class="heading">{getHeading(this.heading)}</h1>
+                {this.renderDescription()}
+                <slot name="content" />
```

Comment:
why this `slot` has a `name`? Read more here: https://github.com/Lundalogik/lime-elements/pull/1934#issuecomment-1386922939